src/pages/game-development/unity/index.md

```diff
@@ -18,7 +18,7 @@ Additionally, Unity supports the low-level APIs Metal on iOS and macOS and Vulka
 
 For 3D games, Unity allows specification of texture compression and resolution settings for each platform that the game engine supports, and provides support for bump mapping, reflection mapping, parallax mapping, screen space ambient occlusion (SSAO), dynamic shadows using shadow maps, render-to-texture and full-screen post-processing effects.
 
-Unity also offers services to developers, these are: Unity Ads, Unity Analytics, Unity Certification, Unity Cloud Build, Unity Everyplay, Unity IAP, Unity Multiplayer, Unity Performance Reporting and Unity Collaborate.
+Unity also offers services to developers, these are: Unity Ads, Unity Analytics, Unity Certification, Unity Cloud Build, Unity Everyplay, Unity IAP, Unity Multiplayer, Unity Performance Reporting and Unity Collaborate. Besides this, Unity has an asset store where developer community can download and upload both commercial and free third party resources such as textures, models, plugins, editor extensions and even entire game examples.
```

Comment:
You should change "...where developer community" to "where **the** developer community."